ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -285,7 +285,7 @@ private Response handlePushQuery(
         disconnectCheckInterval.toMillis(),
         objectMapper);
 
-    log.info("Streaming query '{}'", statement.getStatementText());
+    log.info("Streaming query '<retracted>'");
```

Comment:
I'm wondering if it'd be better to introduce the QueryMask changes onto this branch, to avoid needing changes like this. Is your plan to introduce the QueryMask for INSERT VALUES statements onto older branches in a later PR, or just newer ones?

Reply:
The agreed plan is to introduce fine-grained masking only on latest versions (master and 7.x) and use the existing hotfix logic hiding the statements/exceptions on older ones

Reply:
If that's the case, then there are a bunch of other places which will need to be updated, including https://github.com/confluentinc/ksql/blob/8036a52baa62e7f279d0b2aadca342525708e742/ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java#L202

Reply:
I don't have context on "the existing hotfix logic" but it sounds like there may have been gaps?